packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -281,6 +281,13 @@ export class DataStores implements IDisposable {
             return false;
         }
 
+        const currentHandle = new FluidObjectHandle(
```

Comment:
nit: currentHandle -> aliasedHandle or just handle. The name currentContext is also a little off. I would call it aliasedContext or just context.